i18n/en.json

```diff
@@ -982,6 +982,34 @@
     "id": "api.command_shrug.name",
     "translation": "shrug"
   },
+  {
+    "id": "api.command_mute.desc",
+    "translation": "Mutes the notifications for this channel"
```

Comment:
Please update to: `Turns off desktop, email and push notifications for the current channel or the [channel] specified.`

Reply:
And please update the slash command title to `/mute [channel]`, 

![image](https://user-images.githubusercontent.com/14333569/33141697-2cb3a460-cf71-11e7-81aa-697d6608515c.png)

Similar to the format of other slash commands, for example:
![image](https://user-images.githubusercontent.com/14333569/33141736-4c44c534-cf71-11e7-8475-ce7ff1e0941c.png)